pages/index.tsx

```diff
@@ -1,25 +1,65 @@
 import type { NextPage } from "next";
-import React, { useState } from "react";
-import { EuiText, EuiFieldText, EuiFlexGroup, EuiFlexItem } from "@elastic/eui";
+import React, { useState, useEffect } from "react";
+import { EuiText, EuiFieldText, EuiFlexGroup, EuiFlexItem, EuiButton } from "@elastic/eui";
+import { useRouter } from 'next/router';
 
 const Home: NextPage = () => {
-  const [room, setRoom] = useState("");
+  const router = useRouter();
+  const [newRoom, setNewRoom] = useState("");
+  const [joinRoom, setJoinRoom] = useState('');
+
+  useEffect(() => {
+    if(!newRoom) {
+      setNewRoom(roomGen());
+    }
+  }, [newRoom]);
+
+  const roomGen = () => {
+    function a() {
+      return String.fromCharCode(65+Math.floor(Math.random() * 26));
```

Comment:
Nice! This is neat

Reply:
Yeah. There's probably a way to clean up the Concat part of this, but we can worry about that later.

Reply:
++ to symbols. I can type those and letters